environments/kubernetes/jaeger.libsonnet

```diff
@@ -1,6 +1,6 @@
 (import '../../components/jaeger-collector.libsonnet') + {
   jaeger+:: {
     namespace:: 'observatorium',
-    image:: 'jaegertracing/all-in-one:1.14.0',
+    image:: 'jaegertracing/all-in-one:1.15.0',
```

Comment:
1.15.1 is available. Perhaps 1.15, so that the latest 1.15 is used?

Reply:
We should always *fully* specify the semver of the images so that deployments are fully reproducible and will always run the same containers.

Reply:
That's fair.